WordPress/Classes/Models/BasePost.h

```diff
@@ -32,9 +39,40 @@ typedef enum {
 @property (nonatomic) AbstractPostRemoteStatus remoteStatus;
 @property (nonatomic, strong) NSNumber * post_thumbnail;
 
-@property (nonatomic, assign) BOOL isFeaturedImageChanged;
+// Helpers
+@property (nonatomic, strong) NSString *pathForDisplayImage; // Cached path of an image from the post to use for display purposes. Not part of the post's canoncial data.
+@property (nonatomic, assign) BOOL isFeaturedImageChanged; // BOOL flag if the feature image was changed.
 
-- (NSArray *)availableStatuses;
+/**
+ Returns the localized title for the specified status.  Status should be
+ one of the `PostStatus...` constants.  If a matching title is not found
+ the status is returned.
+
+ @param string The post status value
+ @return The localized title for the specified status, or the status if a title was not found.
+ */
++ (NSString *)titleForStatus:(NSString *)status;
+
+/**
+ Transforms the specified string to plain text.  HTML markup is removed and HTML entities are decoded.
+
+ @param string The string to transform.
+ @return The transformed string.
+ */
++ (NSString *)makePlainText:(NSString *)string;
+
+/**
+ Create a summary for the post based on the post's content.
+
+ @param string The post's content string. This should be the formatted content string.
+ @return A summary for the post.
+ */
++ (NSString *)createSummaryFromContent:(NSString *)string;
```

Comment:
Rather minor, feel free to ignore if you disagree: I'd follow the style of the other method names such as `titleForStatus` and name this method `summaryFromContent` instead.